src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -407,6 +407,10 @@ void ImageReplayer<I>::handle_bootstrap(int r) {
   } else if (r == -EEXIST) {
     on_start_fail(r, "split-brain detected");
     return;
+  } else if (r == -ENOENT) {
```

Comment:
Is it intentional to have this change in "qa/rbd-mirror: add multipeer tests" commit?

Reply:
I included with the qa change because this allow to check this case and it's not changing anything besides the error message. With the new bootstrap behavior we no longer return "EREMOTEIO" anymore (I removed this status message in the first commit).

So yes it's intentional but I can move it elsewhere if you think it's appropriate.

Reply:
I think it should be moved eventually but you may leave it here for now.